From rik.wade@theplanet.net Fri Jan 14 16:22:07 2000
Envelope-to: alan@lxorguk.ukuu.org.uk
Delivery-date: Fri, 14 Jan 2000 16:22:07 +0000
From: rik.wade@theplanet.net
Date: Fri, 14 Jan 2000 17:19:26 +0000
To: alan.cox@linux.org
Subject: patch (traceroute) 1.4a5
Message-ID: <20000114171926.X953@theplanet.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=us-ascii
X-Mailer: Mutt 1.0i

Alan,

Please find below a small patch for traceroute 1.4a5 which allows it to 
trace to the domain:

llanfairpwllgwyngyllgogerychwyrndrobwllllantysiliogogogoch.co.uk

The current limit is hard-coded to 64bytes. According to the RFC
shouldn't it be 256 max for the entire name?

Anyway, a simple patch which should work:


thanks,
rik wade

--- traceroute-1.4a5/traceroute.c.hostname	Fri Jan 14 12:34:50 2000
+++ traceroute-1.4a5/traceroute.c	Fri Jan 14 12:42:33 2000
@@ -240,3 +240,3 @@
 #ifndef MAXHOSTNAMELEN
-#define MAXHOSTNAMELEN	64
+#define MAXHOSTNAMELEN	256
 #endif
-- 
+------------------------------------------------------------------------+
| rik.wade@theplanet.net                          mob: +44(0)7771 766621 |
| Internet Systems Developer                      tel: +44(0)1132 076054 |
| Planet Online Limited                           fax: +44(0)1132 346065 |
+------------------------------------------------------------------------+
